scripts/verify-binary-compatibility

```diff
@@ -5,58 +5,8 @@ set -e -o pipefail -u
 self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
 
-# Note: this should be kept in sync with build.gradle -> patchPluginXml.
-# We're not including the release that we're currently building against (see build.gradle -> allprojects.ext.intellijVersion)
-# since for this case binary compatibility is essentially guaranteed by the compiler.
-# Releases list: https://www.jetbrains.com/intellij-repository/releases/ + Ctrl-F for .idea
-intellij_versions=(193.5233.102 201.6668.121)
-
-verifier_version=1.241
-plugin_version=$(extract_version_from_current_wd)
-plugin_zip=$(ls -1t build/distributions/git-machete-intellij-plugin-${plugin_version}*.zip | head -1)
-
-[[ -f $plugin_zip ]] || die "File $plugin_zip not found, run 'buildPlugin' Gradle task"
-
-# See https://github.com/JetBrains/intellij-plugin-verifier
-verifier_dir="$HOME/.cache/intellij-verifier"
-mkdir -p $verifier_dir
-
-verifier_jar_path=$verifier_dir/verifier-all.jar
-if ! [[ -f $verifier_jar_path ]]; then
-  echo "Downloading IntelliJ Plugin Verifier v${verifier_version}..."
-  curl -fsL -o $verifier_jar_path https://dl.bintray.com/jetbrains/intellij-plugin-service/org/jetbrains/intellij/plugins/verifier-cli/${verifier_version}/verifier-cli-${verifier_version}-all.jar
+export JAVA_HOME=$(which java | xargs readlink --canonicalize | xargs dirname | xargs dirname)
```

Comment:
Twice `dirname` to strip `/bin/java` from the end

Reply:
maybe drop the comment there?

Reply:
It's anyway removed on a subsequent PR, not worth it (instead, `JAVA_HOME` is set upfront in the Dockerfile)